src/edge-config.ts

```diff
@@ -0,0 +1,57 @@
+import { createClient, EdgeConfigClient } from '@vercel/edge-config';
+
+export default class EdgeConfigStoreService {
+  public _baseUrl = 'https://api.vercel.com/v1/edge-config/';
+  public client: EdgeConfigClient;
+
+  /**
+   *
+   * @param edgeConfig edge config store connection string
+   * @param edgeStoreId edge config store id
+   * @param vercelApiToken  vercel api token, used for store write operations
+   */
+  public constructor(
+    edgeConfig: string,
```

Comment:
consider naming this something like  `edgeConfigConnString` (I would have had no idea were it not for the jsdoc above)

Reply:
Vercel, after creation of Edge Config Store, creates an env var, EDGE_CONFIG which is needed here.
https://vercel.com/docs/storage/edge-config/get-started

So I would still call this edgeConfig